include/mrdox/Metadata/TemplateArg.hpp

```diff
@@ -0,0 +1,34 @@
+//
+// This is a derivative work. originally part of the LLVM Project.
```

Comment:
You should delete this line because the file is not a derivative. It is a new creation.